packages/client/src/rpc/modules/engine/CLConnectionManager.ts

```diff
@@ -115,11 +115,11 @@ export class CLConnectionManager {
       maximumFractionDigits: 1,
     })
 
-    if (this.config.chainCommon.gteHardfork(Hardfork.MergeForkIdTransition)) {
+    if (this.config.chainCommon.gteHardfork(Hardfork.MergeNetsplitBlock)) {
```

Comment:
I believe this should be Paris. Will check (currently reviewing from mobile) when at home. But I believe CL will not start on mainnet?

Reply:
Why would this be the case?  The only change here is renaming `MergeForkIdTransition` to `MergeNetsplitBlock`.

Reply:
Good point, this should not have impact.